packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.doc.tsx

```diff
@@ -44,5 +44,6 @@ export const DropdownPageProps: IDocPageProps = {
   dos: require<string>('!raw-loader!office-ui-fabric-react/src/components/Dropdown/docs/DropdownDos.md'),
   donts: require<string>('!raw-loader!office-ui-fabric-react/src/components/Dropdown/docs/DropdownDonts.md'),
   isHeaderVisible: true,
-  allowNativeProps: true
+  allowNativeProps: true,
+  isFeedbackVisible: true
```

Comment:
This is just a nitpick, but can we alphabetize these?

Reply:
yes, will do